internal/appsec/listener/httpsec/http.go

```diff
@@ -117,6 +118,10 @@ func (l *wafEventListener) onEvent(op *types.Operation, args types.HandlerOperat
 		}))
 	}
 
+	if _, ok := l.addresses[ossec.ServerIOFSFileAddr]; ok {
+		ossec.RegisterOpenListener(op, &op.SecurityEventsHolder, wafCtx, l.limiter)
+	}
```

Comment:
Would be best to match the other listeners pattern, and have an `ossec.OSAddressesPresent` function, so the listing of supported addresses is centralized.